packages/ui/src/validators/hooks/useValidatorsList.tsx

```diff
@@ -111,6 +111,15 @@ export const useValidatorsList = () => {
           isActive: activeValidators.includes(address),
           totalRewards: rewardHistory.reduce((total: BN, data) => total.add(data.eraReward), new BN(0)),
           APR: apr,
+          commission: validatorInfo.commission.toNumber() / 10 ** 7,
```

Comment:
Please add a function in `@/common/utils.ts` (next to `clamp`):
```ts
export const perbillToPercent = (perbill: BN) => perbill.toNumber() / 10 ** 7
```

and use it here to clarify the code a bit.